nipype/interfaces/slicer/registration/tests/test_auto_VBRAINSDemonWarp.py

```diff
@@ -1,118 +0,0 @@
-# AUTO-GENERATED by tools/checkspecs.py - DO NOT EDIT
```

Comment:
I don't think this file should have been removed.

Reply:
Sorry, should be fixed now